projects/core/src/personalization/config/personalization-config.ts

```diff
@@ -5,5 +5,9 @@ export abstract class PersonalizationConfig {
       id: string;
       timestamp: string;
     };
+    context?: {
+      slotId?: string;
```

Comment:
we use slot `position` to find slot in page. So, `slotPosition` may be more accurate.